view/next-project/src/components/purchasereports/UploadFileModal.tsx

```diff
@@ -65,9 +70,23 @@ const UplaodFileModal: FC<ModalProps> = (props) => {
 
     setIsLoading(true);
     const formData = new FormData();
+
+    //拡張子取得
+    const uploadFileName = imageFile?.name || '';
+    const Extension = uploadFileName.split('.').pop();
+
+    // 日付取得
+    const date = new Date();
+    const month = date.getMonth() < 10 ? '0' + date.getMonth() : date.getMonth();
```

Comment:
```suggestion
    const month = (date.getMonth() + 1) < 10 ? '0' + (date.getMonth() + 1) : (date.getMonth() + 1);
```
0～11で1～12月を表しているらしいです。
![image](https://github.com/user-attachments/assets/dc7e2540-b82f-46e9-a421-ccbf6becaff1)

Reply:
ありがとうございます
修正しました
[[fix]今月を正しく出力する](https://github.com/NUTFes/FinanSu/pull/871/commits/3692c33699dc8123502015d9d46ef0a58223d290)